src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/Runners/DotNetRunner.cs

```diff
@@ -188,5 +193,30 @@ public void ForceClose()
                 _process.Kill();
             }
         }
+
+        private string GetAspNetCoreFrameworkVersion()
+        {
+            if (TargetFrameworkMoniker.Current == TargetFramework)
+            {
+                return DotNetHost.CurrentAspNetCoreVersionString;
+            }
+            throw new InvalidOperationException($"Unsupported target framework moniker: {TargetFramework}");
+        }
+
+        private string GetNetCoreAppFrameworkVersion()
```

Comment:
Same with this method. Move to an extension method in the template.

Reply:
I've moved all TFM conversion methods to a TargetFrameworkMonikerExtensions type in TargetFrameworkMoniker.cs.